src/libYARP_math/src/math.cpp

```diff
@@ -232,6 +233,14 @@ Vector& yarp::math::operator*=(Vector &a, const Vector &b)
     return a;
 }
 
+Quaternion yarp::math::operator*(const Quaternion& a, const Quaternion& b)
+{
+    return Quaternion(a.w()*b.x() + a.x()*b.w() + a.y()*b.z() - a.z()*b.y(),
```

Comment:
added the reference in the documentation.. i've simply copied the reference reported in the matlab documentation.